framework/docker/cosmos/chain.go

```diff
@@ -379,15 +379,25 @@ func (c *Chain) startAllNodes(ctx context.Context) error {
 	return eg.Wait()
 }
 
+// Stop stops all nodes in the chain without removing them.
 func (c *Chain) Stop(ctx context.Context) error {
```

Comment:
a little unrelated but this change makes the DA node and the Chain Node behave the same way. I.e. when you stop them, the container is still present but stopped, there is a new "Remove" which will remove the container (and volume)